L1Trigger/VertexFinder/python/VertexProducer_cfi.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
-VertexProducer = cms.EDProducer('VertexProducer',
+l1tVertexProducer = cms.EDProducer('VertexProducer',
```

Comment:
This file has no duplicates. But certainly renaming it (e.g.) l1tVertxProcer_cfi.py would give it a less generic name

Reply:
Done